### ISSUE TITLE
telephone number input control example is missing programatic labels

### ISSUE THREAD

**Issue by mfairchild365** [angular/material2#15024]
#### What is the expected behavior?
The purpose of each text input in the 'telephone number input control' example must be conveyed to users of screen reader software.

#### What is the current behavior?
The purpose of each text input in the 'telephone number input control' example is not conveyed to users of screen reader software.

#### What are the steps to reproduce?
1. Using either of the following screen readers
   - NVDA 2018.4.1 + Firefox 64
   - VoiceOver for IOS (12.1.3) + Safari
2. Open the [form field example](https://material.angular.io/components/form-field/examples)
3. Navigate to the inputs under the 'Form field with custom telephone number input control' example.
   - In NVDA you can press the tab key until you reach an input
   - In VoiceOver for iOS you can swipe right until you reach an input
4. Observe that the purpose of each of the 3 text inputs is not announced by the screen reader. They are announced as something like 'edit has auto complete'.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
This issue can also be found by using [axe](https://www.deque.com/axe/).
